extension/io.go

```diff
@@ -0,0 +1,17 @@
+package extension
+
+import (
+	"fmt"
+	"os"
+	"path"
+)
+
+const extensionDirName = ".optimus"
```

Comment:
Got it. I will change it to `.optimus/extensions` later. This directory will be stored in the user's home directory. Or in other words, it will be in `~/.optimus/extensions`.

Reply:
It is updated as suggested